pages/technology/[slug].tsx

```diff
@@ -46,7 +46,7 @@ export default function Post({ post, posts, preview }) {
             <article>
               <Head>
                 <title>
-                  {`${post.title} | Next.js Blog Example with ${CMS_NAME}`}
+                  {`${post?.title} | Next.js Blog Example with ${CMS_NAME}`}
```

Comment:
Great! Build passed! Last change! Please change this to  {`${post?.title} | <blog Title>`}